src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -584,22 +584,35 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     });
   };
 
-  zoomOnCurrentAsset = async (currentAsset: Element | null) => {
-    if (!currentAsset || !this.pinchZoomInstance) {
+  handleSearchResult = (foundElement: Element | null) => {
+    this.zoomOnElement(
+      foundElement,
+      this.props.searchZoom || this.props.initialZoom
+    );
+  };
+
+  /**
+   * @param element - any annotation element on SVG, typically asset or document
+   * @param zoomLevel
```

Comment:
This line doesn't really add anything

Reply:
no, it doesn't. But you can't omit jsdoc params where you don't add a description (eslint usually doesn't like it). 

And I don't like verbose descriptions like "zoomLevel is zoom level"

Reply:
Ah ok, fine by me then